ksql-functional-tests/src/main/java/io/confluent/ksql/test/tools/TopologyTestDriverContainer.java

```diff
@@ -58,4 +60,8 @@ TopologyTestDriver getTopologyTestDriver() {
   public Topic getSinkTopic() {
     return sinkTopic;
   }
+
+  public Set<String> getSourceTopicNames() {
+    return sourceTopics.stream().map(Topic::getName).collect(Collectors.toSet());
```

Comment:
nit: do this in the constructor to only create the set once instead of every time we call `getSourceTopicNames`